pages/task/read/views/models/pil.jsx

```diff
@@ -14,11 +14,16 @@ import Modules from '../../../../profile/read/views/modules';
 export default function PIL({ task, values }) {
   const profile = useSelector(state => state.static.profile);
   const establishment = useSelector(state => state.static.establishment);
-  const pil = task.data.action === 'update-conditions' ? values : task.data.data;
+  let pil = task.data.action === 'update-conditions' ? values : task.data.data;
```

Comment:
Can you move this line down so it sits alongside the `if (isReview)`? 👇

Reply:
```suggestion
```

Reply:
I sure can